ReSource.asm

```diff
@@ -45117,7 +45117,14 @@ lbC024FBC	lea	(USP.MSG1-ds,a6),a0
 	move.b	(a0)+,(a4)+
 	bra.w	oper20
 
-oper28	btst	#$10,d5
+add_68060_comment
```

Comment:
Hi Philippe, this will add a '68060' comment also for an illegal register number with bit #3 set.
Please put the check at the end of the first routine and jump there from the second.

Reply:
You’re right.
But it can not put at the check at the end of the first routine as it can return from the first or second routine and have an illegal register.
I inserted an illegal register check in add_68060_comment routine.

Reply:
Sorry to bother again but I don't like to put the comment in front of the routine.
oper28 has 3 exit points. Two for ea,reg and reg,ea and one for illegal register. You can unite the first to exit points and put there the check for the 68060 regs and add the comment. That will result in less code.